resources/views/admin/all_restaurants.blade.php

```diff
@@ -0,0 +1,77 @@
+@extends('layouts.app')
+
+@section('styles')
+<!-- Bootstrap -->
+<link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.0/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-9ndCyUaIbzAi2FUVXJi0CjmCapSmO7SnpJef0486qhLnuZ2cdeRhO02iuK6FUUVM" crossorigin="anonymous">
```

Comment:
We can remove the import of bootstrap here because laravel has a bootstrap already installed.